src/config-file.c

```diff
@@ -130,20 +130,43 @@ static gboolean get_key_bool(GKeyFile *key_file, const gchar *group,
         return FALSE;
 }
 
-static gboolean get_key_int(GKeyFile *key_file, const gchar* group, const gchar* key, gint* value, const gint default_value, GError **error)
+/**
+ * @brief Get integer value from key_file for key in group, default_value must
+ * be specified, returned in case key not found in group.
+ *
+ * @param[in]  key_file      GKeyFile to look value up
+ * @param[in]  group         A group name
+ * @param[in]  key           A key
+ * @param[out] value         Output integer value
+ * @param[in]  default_value Return this value in case no value found
+ * @param[out] error         Error
+ * @return FALSE on error (not found is no error; error is set), TRUE otherwise
```

Comment:
a bit misleading having both statements in the same bracket, but I think I got what you meant

Reply:
Agreed. I improved the doc string.